server/config/env.js

```diff
@@ -1,6 +1,6 @@
 const oneSecondMS = 1000;
 
-const mongoUrl = process.env.MONGODB_URI || 'mongodb://localhost:27017/distribute'
+const mongoUrl = 'mongodb://mongo:27017/distribute'
```

Comment:
I would leave the ability to take an env var here. This is the kind of thing we'll want to have configurable, so that we don't have to change hard-coded strings mixed through out the codebase when deploying to different envs vs running locally

Reply:
kk that makes sense